### ISSUE TITLE
Update gifs and other animated images to have ability to pause

### ISSUE THREAD

**Issue by jnm2377** [carbon-design-system/carbon-website#2683]
## Acceptance criteria
**Overview**
- For a11y (WCAG criteria 1.4.2) purposes all animated images need the ability to pause or only play once and then not re-play/loop after that. See below comments from @mbgower about a11y rules around this.
- We can accomplish this by using the [Gatsby theme GifPlayer component](https://gatsby-theme-carbon.vercel.app/components/GifPlayer).

- We need to update any current gifs/videos on our website to have a pause feature.
  - Access every place where we have a gif/video on our website.
  - Create a static image for each gif/video if it does not have one already.
  - Fix any gifs that are visually jumping.

---

Related: #2640 

```[tasklist]
### Tasks
- [ ] https://github.com/carbon-design-system/carbon-website/issues/3961
- [x] Giphys: collect all visuals of where gifs show up on the website from the audit and make the cover art
- [ ] https://github.com/carbon-design-system/carbon-website/issues/3962
```

**Comment by aubrey-oneal** [comment]
WCAG guideline:
 https://www.ibm.com/able/requirements/requirements/?version=v7_2#2_2_2

Moving, blinking, scrolling: 
For any moving, blinking, or scrolling information that
starts automatically, is presented in parallel with other content, lasts more than five seconds, there is a mechanism for the user to pause, stop, or hide it;

**Comment by mbgower** [comment]
>  Reach out to @mbgower if it is length dependent. For example: if a video/gif is under a certain amount of time, do we need a pause button? Is WCAG criteria 1.4.2 the correct number?
 
Yes, @aubrey-oneal correctly linked this to 2.2.2, which is any autoplaying content that lasts more than 5 seconds (the original issue description talks about 1.4.2, which only covers audio).

So, a few possible techniques... 

**An ability to pause or hide is one method.**
I'm sure folks are familiar with slack, where you can hide any gif (animated or otherwise) someone puts in your thread simply by collapsing them (slack implementa each with its own expand/collapse control).
Another way would be to cause items to stop when they take keyboard focus. You don't actually have to provide a way of restarting them for accessibility, but you may decide to make this be "pause" instead of "stop", and provide a mechanism to replay.
Another way would be to provide a Stop all animations mechanism on a page, which would then allow you to not bother with a granular pause mechanism per animation. There are some advantages to this (only one control needed to activate). But maybe you _want_ to provide that granular control in a training scenario?

**Not have animations run for more than 5 seconds.**
Obviously if you just make all animations be a maximum of 5 seconds, you never fail the SC
There is potentially more scope for variation here than one might imagine. If you wanted to have some kind of looping animation, you could cycle it less frequently. For instance, a 5 second animation could cycle every 10 seconds. that would meet the letter of the requirement, although I have no idea on how disruptive folks would find that. Sounds like a research questions.

**Don't make things start automatically**
You could trigger them on hover and focus. And turn them off when items lose focus and hover. I've always thought that is a relatively elegant way of solving this. It would help to have some kind of icon to indicate they are little videos/animations so users knew to trigger them.

**Comment by tay1orjones** [comment]
We have the GifPlayer now, so this issue is just for wrapping the .gif's we have on the website in a GifPlayer. There's ~24 gifs on the website right now, [here's one example](https://carbondesignsystem.com/components/data-table/usage#searching)

**Comment by thyhmdo** [comment]
Closed this issue. Follow up with [this new one](https://github.com/carbon-design-system/carbon-website/issues/4048).
